packages/devtools_app/lib/src/screens/memory/panes/leaks/LEAK_TRACKING.md

```diff
@@ -9,43 +10,64 @@ This page and functionality are under construction. See https://github.com/flutt
 
 ### Glossary
 
-**GC**: garbage collection.
+**GC**: garbage collection. The process of reclaiming memory that is no
+longer being used.
 
-**Memory Leak**: progressive usage of more and more memory by an application, for example repetedly creating and not disposing a listener.
+**Memory Leak**: progressive use of more and more memory by an application,
+for example, by repeatedly creating (but not disposing of) a listener.
 
-**Memory Bloat**: use of more memory than is necessary for optimal performance, for example using too large images or not canceling a stream.
+**Memory Bloat**: using more memory than is necessary for optimal performance,
+for example, by using overly large images or not canceling a stream.
 
-**Object's Retaining Path**: path from the object to a root object that prevents the object from being GCed.
+**Object's Retaining Path**: references to an object from a root object that
+prevents the object from being GCed.
 
 
 ### Leak Types
 
-To detect memory leaks, the tool uses the fact that, with proper memory management, a dart object disposal and GC events should happen sequentially, close to each other.
+To detect memory leaks, the tool assumes that, with proper memory management,
+an object's disposal and collection should happen sequentially, close to each other.
```

Comment:
The word 'collection' has too many meanings. Let's say 'garbage collection'.

Reply:
ok, sounds good